src/components/Pages/Resume/index.js

```diff
@@ -74,15 +73,28 @@ export default function Resume() {
                     <br />
                     <h4>RIDER</h4>
                     <p>Foodora.Co | 2018 - 2019</p>
+                    <h5>Delivering your pizza on a rainy winter night</h5>
```

Comment:
lol be creative not just copy my shit .. but ok :p

Reply:
maaaaaa bddi hahahah